bindings/ergo-lib-wasm/src/transaction.rs

```diff
@@ -74,6 +74,13 @@ impl Transaction {
         JsValue::from_serde(&self.0.clone()).map_err(|e| JsValue::from_str(&format!("{}", e)))
     }
 
+    /// JSON representation
```

Comment:
I just added this for consistency's sake since I was looking through this files while debugging this problem and `UnsignedTransaction` had both but `Transaction` only had `to_json()`